### DISCUSSION TITLE
Histoire and Vite 3

### DISCUSSION THREAD

**Discussion by victorlmneves** [histoire-dev/histoire#209] [Q&A]
There is any prediction for having Histoire ready for Vite 3?
We have plans for upgrading to Vite 3 but meanwhile, I saw someone saying that Histoire is not compatible with Vite 3
Thanks

**Comment by hrishikesh-k** [comment]
It seems to be working fine with Vite 3 for me. I had to run `npm install` with the `--force` flag, but that's about it.

**Reply by victorlmneves** [reply]
Which Histoire version are you using?

**Reply by hrishikesh-k** [reply]
`^0.8.4`

**Reply by victorlmneves** [reply]
@Hrishikesh-K for me it doesn't work :/

**Reply by hrishikesh-k** [reply]
Weird, the latest 0.9.x doesn't seem to work for me too.

**Reply by victorlmneves** [reply]
Meanwhile, I saw that there was already an open issue for making it compatible even before I create this topic :/
https://github.com/histoire-dev/histoire/issues/189
